src/refract/VisitorUtils.cc

```diff
@@ -0,0 +1,43 @@
+//
+//  refract/VisitorUtils.cc
+//  librefract
+//
+//  Created by Vilibald Wanča on 09/11/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+#include "Element.h"
+#include "Visitors.h"
+#include "VisitorUtils.h"
+namespace refract
+{
+
+    IElement* DefaultAttribute(const IElement& e) {
```

Comment:
What is this function about? and what's the difference between this and `GetDefault`?